src/services/axios/auth-api-calls/teams/index.js

```diff
@@ -0,0 +1,18 @@
+import axios from '../../axios'
+import { toast } from 'react-toastify'
+
+export async function joinTeam(team) {
+	return axios.post('teams/' + team.id + '/enrollments')
```

Comment:
Maybe use string literals here? \`teams/${team.id}/enrollments\`

Reply:
Done in https://github.com/jamtasticgd/jamtastic/pull/37/commits/d78f63ba10355c7b4b1bce87fa32a3c205afe6f5